third_party/packages/flutter_svg/pubspec.yaml

```diff
@@ -12,9 +12,9 @@ dependencies:
   flutter:
     sdk: flutter
   http: ^1.0.0
-  vector_graphics: ^1.1.11+1
+  vector_graphics: ^1.1.13
   vector_graphics_codec: ^1.1.11+1
-  vector_graphics_compiler: ^1.1.11+1
+  vector_graphics_compiler: ^1.1.13
```

Comment:
Based on the order things landed, this should be 1.1.14 (`vector_graphics` and `vector_graphics_compiler` ended up with opposite order of WASM support vs relaxed dependencies, it looks like.)

Reply:
you're right, fixed